security/pkg/nodeagent/sds/server.go

```diff
@@ -85,13 +84,8 @@ func (s *Server) Stop() {
 func (s *Server) initWorkloadSdsService() {
 	s.grpcWorkloadServer = grpc.NewServer(s.grpcServerOptions()...)
 	s.workloadSds.register(s.grpcWorkloadServer)
-
 	var err error
 	s.grpcWorkloadListener, err = uds.NewListener(security.WorkloadIdentitySocketPath)
-	if err != nil {
```

Comment:
Why is this err log deleted?

Reply:
This NewListener call it self is only used in tests (to avoid timing issues) and the go routine below logs the same if it is the workloadListener is not setup. So this log line is redundant.